### PR TITLE
Added live debugger support and made fields more simple

### PR DESCRIPTION
**Pull request by vascograce**
* Added support for the debug flag to be passed on
* Add support for allowing for multiple event name mappings 
   - Like GA4 event called "completed_order" to map to bg event "order_completed"
* Made the cookie settings for straight forward, by specifying if the top level domain for the sst container matches the website domain
* Removed unused field event id, as this is being passed from the web tracking and not set on the sst container

### CODE REVIEW

File: template.js [hunk 1]
```diff
@@ -33,11 +35,20 @@ const mappedEventNames = {
   'gtm4wp.orderCompletedEEC': 'checkout_completed'
 };
 
+
+// Determines first or third party context, when serverIsSameSite == true first party
+function getSameSiteAttribute() {
+  if (data.serverIsSameSite == true){
```
Comment (foadbillygrace) [inline template.js:41]: serverIsSameSite is always in the data object?
We don't wanna check if it exists or not?

File: template.js [hunk 2]
```diff
@@ -223,11 +253,6 @@ function mapCustomEventData(eventName, allEventData, data){
       }
     }
   }
-
-  // If any of the values are set, then override anything we currently have
-  if (data.transaction_id) customEventData.transaction_id = data.transaction_id;
-  if (data.value) customEventData.value = data.value;
-  if (data.currency) customEventData.currency = data.currency;
```
Comment (foadbillygrace) [inline template.js:230]: And how we set this values? So we expect this to be in the data object always?

**Review by foadbillygrace — APPROVED** [review]
LGTM!
It make the code more clean.
i only had some small questions!

File: template.js (hunk 1 above)
Comment (vascograce) [reply template.js:41]: Yeah so everything .data are parameters that are set by the fields on the tag itself. And this one is a boolean, so its always true or false

File: template.js (hunk 2 above)
Comment (vascograce) [reply template.js:230]: So originally it was set in a way to only set this once for all events going through this tag, however people need to be able to change this per event. For now I created a table to map all the custom events, and if people want to map these values as well, then i can update the table for that as well!

**Pull request merged**